atom/browser/api/atom_api_app.h

```diff
@@ -70,6 +70,8 @@ class App : public AtomBrowserClient::Delegate,
       std::unique_ptr<CertificateManagerModel> model);
 #endif
 
+  std::string GetAppPath();
```

Comment:
I wonder if this might be better to store as a `base::FilePath`. That is what `App::GetPath` returns and it looks like on Windows `base::FilePath` will be converted to a `std::wstring` and it will be a `std::string` on other platforms.